src/js/components/RangeInput/index.d.ts

```diff
@@ -9,6 +9,7 @@ export interface RangeInputProps {
   name?: string;
   step?: number;
   value?: number | string;
+  inputValue?: boolean;
```

Comment:
Can we move this above `min` just to keep things nice and clean and alphabetical